libnavui-maps/gradle.properties

```diff
@@ -0,0 +1,4 @@
+RELEASE_SDK=ui
+POM_ARTIFACT_ID=ui-v1
```

Comment:
We'll need to update all of those IDs. Unnecessary now though, since there's no release process.

Reply:
I agree